pkg/kvs/kvs.go

```diff
@@ -0,0 +1,34 @@
+package kvs
+
+import (
+	"errors"
+
+	"github.com/ethersphere/bee/pkg/api"
+	"github.com/ethersphere/bee/pkg/kvs/manifest"
+	"github.com/ethersphere/bee/pkg/kvs/memory"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+var ErrInvalidKvsType = errors.New("kvs: invalid type")
```

Comment:
this file feels unnecessary and overcomplicates things

Reply:
removed